.gitlab-ci.yml

```diff
@@ -757,6 +758,8 @@ variables:
       - test/kitchen/test/integration/system-probe-test/**/*
       - test/kitchen/test/integration/win-sysprobe-test/**/*
       - .gitlab/functional_test/system_probe.yml
+      - test/new-e2e/systemProbe/**/*
```

Comment:
This is super picky, but can we use a dash for the directory name (`system-probe`) instead of camelcase? We don't really use camelcase anywhere in filenames/directories.